cpp/velox/compute/VeloxColumnarToRowConverter.cc

```diff
@@ -67,6 +69,16 @@ arrow::Status VeloxColumnarToRowConverter::Init() {
         int64_t bytes = RoundNumberOfBytesToNearestWord(length);
         lengths_[row_idx] += bytes;
       }
+    } else if (arrow::is_list_like(field->type()->id())) {
+      auto arrayVector = vecs_[col_idx]->as<velox::ArrayVector>();
+      auto rawSizes = arrayVector->rawSizes();
+      for (int row_idx = 0; row_idx < num_rows_; row_idx++) {
+        // Calculate the ArrayData size
+        auto numElements = rawSizes[row_idx];
+        auto total_size = 8 + CalculateBitSetWidthInBytes(numElements) +
+            numElements * 8; // TODO: fixed size except decimal and string type
```

Comment:
Need a helper method to recursively compute total size including all sub-elements given we need to support other types (e.g., struct) shortly.